ncard_app/templates/detail_views/add_award.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'events/base.html' %}
+{% load crispy_forms_tags %}
+{% load static %}
+<link rel="stylesheet" href="{% static 'style.css' %}" type="text/css" media="screen" >
+{% block title %}Add Award{% endblock %}
+
+{% block content%}
+    <a class="close-btn" href="{% url 'list-awards' %}">
+        <button type="button" class="btn-close" aria-label="Close"></button>
```

Comment:
The close buttons for add_award.html and edit_award.html are different from the close buttons on other forms. Should they be consistent or is there a good reason for the difference?

Reply:
the add award and edit award close button is the proper one, will update the rest tomorrow